### PR TITLE
jq support for stamping

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

**Review by kormide — COMMENTED** [review]
Very cool 🌮 . Just a few FIXMEs to clean up.

**Comment by alexeagle** [comment]
Which fixmes? I must be missing something 🥸

File: docs/jq.md [hunk 1]
```diff
@@ -86,6 +86,8 @@ jq(
     ],
     filter = "{ deps: split("\n") | map(select(. | length > 0)) }",
 )
+
+# FIXME: mention stamping
```
Comment (kormide) [inline docs/jq.md:90]: Here's one

**Comment by kormide** [comment]
fyi: @jbedard we could use this in the angular cli. I might go back and try this.

**Review by kormide — APPROVED** [review]

**Pull request merged**